packages/scripts/config/jest-e2e.config.js

```diff
@@ -9,6 +9,7 @@ const path = require( 'path' );
 const { hasBabelConfig } = require( '../utils' );
 
 const jestE2EConfig = {
+	testRunner: 'jest-circus/runner',
```

Comment:
Should we update the test runner for unit tests as well for consistency?

Reply:
Worth a shot!

Reply:
Added in https://github.com/WordPress/gutenberg/pull/31178